### ISSUE TITLE
RedShift driver in 4.1.0 does not support SSL keeps asking for sslkey 

### ISSUE THREAD

**Issue by waterytowers** [dbeaver/dbeaver#1899]
Can we add support for SSL connections to RedShift?

It works in DBeaver EE 4.0.5 but not in 4.1.0?

The only settings used in EE 4.0.5 are:
  AuthMech : REQUIRE 
  ssl : TRUE

The same settings do not work in 4.1.0?  It keeps asking to enter something for sslkey.

**Comment by serge-rider** [comment]
There were no any changes in RedShift config in 4.1
So it should work the same way as in 4.0.5.

**Comment by tati-kru** [comment]
There isn't any activity about this ticket very long time. Perhaps, the question was solved. It will be closed. However, if you think that it is actual just let us know.

**Comment by waterytowers** [comment]
Sorry I managed to fix the issue by deleting the driver and adding it again.  I was then very careful in adding only the 2 settings above.

The problem started when I didn't know which settings were required to get SSL working with RedShift. 
 I tried a few different settings to get the driver working with SSL.  I had setup this for a few database tools trying a few to find the best tool for multiple platforms.  I tried similar settings to the other tools and nothing was working.  Having tried a few of the ssl settings DBeaver was always asking for me to enter an sslkey.  I tried resetting all the settings and it was still asking for the sslkey.  Deleting the driver was the only way it would stop asking for the sslkey.

Not sure if that helps?  I expect there is a bug somewhere that doesnt clear the settings properly when your reset to default?

I was at this for a couple of hours and it was very frustrating.  Maybe a simple guide for how to apply appropriate settings for each database for common situations.  Our security team have mandated all database connections use SSL and I would expect this to be a common scenario so expected to find an easy answer online.

My time is limited as I am in a small team so am not likely to be able to come back with more details.  It may have been a problem in both versions I didn't go back and test once it was working.

Thanks
Dave
